search-commons/src/test/java/no/unit/nva/search2/common/QueryToolsTest.java

```diff
@@ -0,0 +1,53 @@
+package no.unit.nva.search2.common;
+
+import static java.util.Objects.nonNull;
+import static no.unit.nva.search2.constant.Words.AMPERSAND;
+import static no.unit.nva.search2.constant.Words.EQUAL;
+import static nva.commons.core.StringUtils.EMPTY_STRING;
+import static nva.commons.core.attempt.Try.attempt;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map.Entry;
+import nva.commons.core.JacocoGenerated;
+
+public final class QueryToolsTest {
```

Comment:
Should this be named `QueryToolsTest`? Does it test anything?

Reply:
No, I originally used `QueryTools`, but that was problematic since it aleady exist in `main`
Do you have a suggestion?